docs/installation/server-config/security.rst

```diff
@@ -1,4 +1,227 @@
+are signed. For more information, see the `Security section of the Kafka documentation <http://kafka.apache.org/documentation.html#security>`__
+and the :ref:`security parameter reference <security-config-ref>`.
+
+
+Using KSQL with a Kafka Cluster Secured with ACLs
+-------------------------------------------------
+
+You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the cluster is
+interactive or non-interactive.
+
+.. tip:: For more information about ACLs see :ref:`kafka_authorization` and for more information about interactive and
+         non-interactive queries, see :ref:`restrict-ksql-interactive`.
+
+-------------------------
+Interactive KSQL clusters
+-------------------------
+
+Kafka clusters that use ACLs to control access to resources require clients to authenticate as a particular user. Interactive
+KSQL clusters require that this authenticated user has open access to create, read, write, delete topics, and use any
+consumer group.
+
+The required ACLs are:
+
+- ``DESCRIBE_CONFIGS`` permission on the ``CLUSTER``.
+- ``CREATE`` permission on the ``CLUSTER``.
+- ``DESCRIBE``, ``READ``, ``WRITE`` and ``DELETE`` permissions on the ``<any>`` ``TOPIC``.
+- ``DESCRIBE`` and ``READ`` permissions  on the ``<any>`` ``GROUP``.
+
+It is still possible to restrict the KSQL user from accessing specific resources using ``DENY`` ACLs. For example, you can add a
+``DENY`` ACL to stop KSQL queries from accessing a topic that contains sensitive data.
+
+----------------------------------------
+Non-Interactive (headless) KSQL clusters
+----------------------------------------
+
+Non-interactive KSQL clusters can be run with much more restrictive ACLs, though it currently requires a bit of effort to
+determine what ACLs are required. This will be improved in future KSQL releases.
+
+Standard ACLs
+    The KSQL user always requires:
+
+    - ``DESCRIBE_CONFIGS`` permission on the ``CLUSTER``.
+    - ``DESCRIBE`` permission on the ``__consumer_offsets`` topic.
+
+Source topics
+    KSQL users require ``DESCRIBE`` and ``READ`` permissions for each source and input topic. The topic should already exist
+    when KSQL is started.
+
+Sink topics
+    KSQL users require ``DESCRIBE`` and ``WRITE`` permissions ror each sink and output topic.
+
+    By default, KSQL will attempt to create any output topics that do not exist. To allow this the KSQL user will require
+    ``CREATE`` permissions on the ``CLUSTER``. Alternatively, topics can be created manual before running KSQL. To determine
+    the list of output topics their required configuration, for example, partition count, replication factor,
+    retention policy, etc, you can run KSQL on open cluster first.
+
+    If you want KSQL to create sink topics, then the KSQL user should be granted ``CREATE`` permission on the ``CLUSTER``. KSQL
```

Comment:
> then the _ksql-user_ should

Reply:
Changed to authenticated KSQL user globally